backend/matchmaking/models.py

```
@@ -13,6 +13,9 @@ class WaitingPlayer(models.Model):
 	assigned_color = models.CharField(max_length=10, null=True, blank=True)
 	assigned_game_id = models.IntegerField(null=True, blank=True)
 
+	@database_sync_to_async
```

Pylint (3.3.7)

[warning] Bad indentation. Found 1 spaces, expected 4

(W0311)